packages/sdk/src/_tests/LinearClient.test.ts

```diff
@@ -66,4 +66,23 @@ describe("LinearClient", () => {
       expect(error.type).toEqual(LinearErrorType.AuthenticationError);
     }
   });
+
+  it("supports pagination", async () => {
+    const client = new LinearClient({ apiKey: MOCK_API_KEY, apiUrl: ctx.url });
+    const team = await client.team("someTeamId");
+
+    ctx.customSpecs['endCursorId'] = {
```

Comment:
Feels a bit weird to define it on the context like this. Is there not a way to mock the function instead after the first call?

Reply:
Did a bit of refactor in the test server. The Express route is now declared once at initialization, and `MockContext.res` is updated to let tests provide a function that dynamically returns a `MockSpec` rather than a static one.